canonical-websites/cn.ubuntu.com
# Update the navigation to match the latest design

## Done
Update the navigation to match the latest design

## QA
- Load the demo
- Check the navigation looks like https://www.ubuntu.com
- Check all the different screen sizes

## Issue / Card
FIxes https://github.com/canonical-websites/cn.ubuntu.com/issues/246
